### ISSUE TITLE
Suggesting correct use of grammar in description of OpenTek.

### ISSUE THREAD

**Issue by MuditWadhwa** [Opentek-Org/opentek#79]
![grammarimage](https://user-images.githubusercontent.com/69974521/135598080-51104b04-e7a1-4560-8664-f05781f4ae93.jpeg)
In this image, the last sentence can be improved by better use of  relative pronouns as well as ." Long term goal is to build a community who truly believe in Open Source and building in public." should be replaced by  
"Long term goal is to build a community that truly believes in Open Source and building in public."

**Comment by MuditWadhwa** [comment]
Please assign this issue to me and add the hacktoberfest tag to it.

**Comment by AnirudhPanda** [comment]
Hey @MuditWadhwa 
I have one task for you!!

Check the whole content of the website and suggest the improvement in grammar or speech.
this way it will be a helpful contribution for us too!

Would you like to do it?

**Comment by MuditWadhwa** [comment]
Yes....Definitely...I would give my best. Please assign this this issue to me and add hacktoberFest  tag to it.
